### PR TITLE
fix: add missing tags

### LINKED ISSUE #235: Lack of Scientific domain, Organisation for Services in All Catalogs
All Catalogs
image

Services
image

### PR DESCRIPTION
**Pull request by danielkryska**
closes #235

### CODE REVIEW

**Comment by danielkryska** [comment]
image

**Comment by danielkryska** [comment]
image

**Comment by danielkryska** [comment]
image

**Comment by danielkryska** [comment]
image

**Comment by danielkryska** [comment]
image

File: ui/apps/ui/src/app/collections/data/all/adapter.data.ts [hunk 1]
```diff
@@ -64,8 +64,18 @@ export const allCollectionsAdapter: IAdapter = {
       },
       {
         label: 'DOI',
-        value: data?.url || [],
-        filter: 'url',
+        value: data?.doi || [],
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/collections/data/all/adapter.data.ts:67]: @agpul `DOI` occurs only for Openaire-type resources. What do we want to do with trainings and services?
Do we want to present nothing or a `URL`?

File: ui/apps/ui/src/app/collections/data/all/adapter.data.ts [hunk 2]
```diff
@@ -64,8 +64,18 @@ export const allCollectionsAdapter: IAdapter = {
...
+        filter: 'doi',
+      },
+      {
+        label: 'Scientific domain',
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/collections/data/all/adapter.data.ts:71]: @agpul  `Scientific domain` is only for services! For OAG-type resources, there is `fos` (field of science) which looks similar at least to me. Please look at this filter [Explore](https://explore.openaire.eu/search/find?resultbestaccessright=%22Open%2520Access%22&active=result). Maybe we can present `Scientific domain` for services and `fos` for OAG? For trainings most likely there is nothing like this

**Review by Michal-Kolomanski — COMMENTED** [review]
The code looks good but I have a few business concerns. Before merging it also be safer, to change aliases to the latest data to enable testing on the PR instance new feature (as I was writing)

File: ui/apps/ui/src/app/collections/data/all/adapter.data.ts (hunk 1 above)
Comment (agpul) [reply ui/apps/ui/src/app/collections/data/all/adapter.data.ts:67]: nothing:)

File: ui/apps/ui/src/app/collections/data/all/adapter.data.ts (hunk 2 above)
Comment (agpul) [reply ui/apps/ui/src/app/collections/data/all/adapter.data.ts:71]: in the future, this field will be also valid for training, Scientific domain = FOS from my perspective, and in all catalogs it should be visible as one filter

File: ui/apps/ui/src/app/collections/data/all/adapter.data.ts (hunk 1 above)
Comment (Michal-Kolomanski) [reply ui/apps/ui/src/app/collections/data/all/adapter.data.ts:67]: ok so it's good as it is

File: ui/apps/ui/src/app/collections/data/all/adapter.data.ts (hunk 2 above)
Comment (Michal-Kolomanski) [reply ui/apps/ui/src/app/collections/data/all/adapter.data.ts:71]: ok, so this should be done @danielkryska

Comment (agpul) [reply ui/apps/ui/src/app/collections/data/all/adapter.data.ts:71]: #279 - read and comment on what you think. I've created another ticket to resolve it as a new issue.

Comment (Michal-Kolomanski) [reply ui/apps/ui/src/app/collections/data/all/adapter.data.ts:71]: ping @danielkryska

**Review by agpul — APPROVED** [review]

**Review by Michal-Kolomanski — COMMENTED** [review]
@agpul I am also wondering if we should color our DOIs in some way. Currently, it is black as everything. For me, it is not obvious that is an interactive link to the resource. Look how OpenAire is presenting DOIs - as blue links
image

**Review by Michal-Kolomanski — APPROVED** [review]

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**